### PR TITLE
Block audit

### PR DESCRIPTION
**Pull request by hansmosh**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,47 +1,68 @@
...
+--------
 None
 
-Input
--------
-Input signals will trigger REST polls.
-
 Getting API Access
----------
+------------------
```
Comment (cowleyk) [inline README.md:27]: `Getting API Access` section is duplicated after each block.  Probably only needs to be at bottom of README

File: README.md [hunk 2]
```diff
@@ -1,47 +1,68 @@
...
 Properties
---------------
- * **API Key** - This is your API token returned by Moj.io. See the [Getting API Access](#getting-api-access) section for details
+----------
+- **creds**: Credentials to connect to Mojio API.
+- **include_query**: Whether to include queries in request to Mojio.
+- **polling_interval**: How often Mojio is polled. When using more than one query. Each query will be polled at a period equal to the *polling interval* times the number of queries.
+- **queries**: Inherited from base rest polling block but not used.
+- **retry_interval**: When a url request fails, how long to wait before attempting to try again.
+- **retry_limit**: Number if times to retry on a poll.
```
Comment (cowleyk) [inline README.md]: *`Number of times...`

File: README.md (hunk 1 above)
Comment (hansmosh) [reply README.md:27]: We don't really have a way to do this though...

Comment (hansmosh) [reply README.md:27]: At least with `nio buildreadme`

**Pull request merged**